draft-ietf-tls-tls13.md

```diff
@@ -1051,14 +1047,22 @@ fragment
 : The AEAD encrypted form of TLSPlaintext.fragment.
 {:br }
 
-Each AEAD cipher suite MUST specify how the nonce supplied to the AEAD
-operation is constructed, and what is the length of the
-TLSCiphertext.nonce_explicit part. In many cases, it is appropriate to use
-the partially implicit nonce technique described in Section 3.2.1 of
-{{RFC5116}}; with record_iv_length being the length of the explicit part. In
-this case, the implicit part SHOULD be derived from key_block as
-client_write_iv and server_write_iv (as described in {{key-calculation}}), and
-the explicit part is included in GenericAEAEDCipher.nonce_explicit.
+
+The nonce for the AEAD construction is formed as follows:
+
+  1. iv_length is set to max(8 bytes, N_MIN) for the AEAD algorithm
+     (see {{RFC5116}} Section 4).
+
+  2. The 64-bit sequence number is padded to the left with zeroes
+     to iv_length and XORed with the fixed iv, which is of length
+     iv_length.
+
```

Comment:
"the fixed iv" is unclear here. I assume you mean the "the static client or server iv" (or otherwise worded).